src/components/Shared/EmptyList.jsx

```diff
@@ -0,0 +1,48 @@
+import { Box, Typography, Button } from "@mui/material";
+
+//for My Planner Page
+{/* <EmptyList
+  icon={<CalendarTodayIcon sx={{ fontSize: 200 }} />}
+  message="Your planner is empty right now, but that's okay—it's just waiting for you to fill it with your exciting events!"
+  buttonText="Explore Events >>"
+/>; */}
+
+//for Liked List Page
+{
+  /* <EmptyList
+  icon={<FavoriteIcon sx={{ fontSize: 200 }} />}
+  message="No events liked yet? Start liking events, and they'll appear here for easy access later!"
+  buttonText="Explore Events >>"
+/>; */
+}
+
+//for Event Search Page
+{
+  /* <EmptyList
+  icon={<FindInPageIcon sx={{ fontSize: 200 }} />}
+  message="Sorry, we couldn't find any events matching your search criteria. Try adjusting your filters or search again later!"
+  buttonText="Explore More Events >>"
+/>; */
+}
+
+const EmptyList = ({ icon, message, buttonText, onClick }) => {
```

Comment:
Nihal your PR looks good for me, I like the code structure and the comments you provide are really helpful to understand and implement the component. The only issue I noticed: the onClick prop is defined in the function signature but isn't used in the Button.

Reply:
You're correct that it hasn’t been implemented yet. Since this is a reusable component only, the onClick prop will be connected by the parent component later. Thanks again for reviewing!